docs/dev/tutorials/custom-paymaster-tutorial.md

```diff
@@ -85,7 +89,7 @@ Note, that only the [bootloader](../developer-guides/contracts/system-contracts.
 
 In this tutorial, we want to charge the user one unit of the `allowedToken` in exchange for her fees being paid by the contract.
 
-The input that the paymaster should receive is encoded in the `paymasterInput`. As described [here](../developer-guides/transactions/aa.md#built-in-paymaster-flows), there are some standardized ways to encode user interactions with paymasterInput. To charge the user, we will require that she has provided enough allowance to the paymaster contract. This is what the `approvalBased` flow can help us with.
+The input that the paymaster should receive is encoded in the `paymasterInput`. As described [in the paymaster documentation](../developer-guides/transactions/aa.md#built-in-paymaster-flows), there are some standardized ways to encode user interactions with paymasterInput. To charge the user, we will require that she has provided enough allowance to the paymaster contract. This is what the `approvalBased` flow can help us with.
```

Comment:
just remember to remove the /transaction/ path

Reply:
fixed